src/views/Home/Home.css

```diff
@@ -0,0 +1,69 @@
+.content {
+  width: 100%;
+  position: absolute;
+  top: 0;
+  bottom: 96px;
+}
+
```

Comment:
before we merge we need to make it responsive so its pending

Reply:
Kindly provide the designs for the mobile view for me to proceed

Reply:
please check the slack channel 

https://www.figma.com/file/s7kKafXNQ6bu8LF8IIIi3H/UGRO-MOBILE-00?node-id=0%3A1

Reply:
Noted
I will update you shortly